ChangeLog

```diff
@@ -10,6 +10,10 @@ Changes
    * Remove some redundant code in bignum.c. Contributed by Alexey Skalozub.
    * Support cmake build where Mbed TLS is a subproject. Fix
      contributed independently by Matthieu Volat and Arne Schwabe.
+   * Allow configuring the prefix operator for the archiver tool when compiling
```

Comment:
“The archiver tool” is more commonly known as `ar`, and the main if not sole reason for this change is LLVM ar. Suggestion:

> Add an option in the makefile to support ar utilities where the operation letter must not be prefixed by "-", such as LLVM. Found and fixed by Alex Hixon.

Reply:
I will modify the ChangeLog when it is decided whether the change for DLEXT should be reverted.